src/mmw/js/src/modeling/templates/scenarioTabPanel.html

```diff
@@ -6,7 +6,7 @@
             <i class="fa fa-caret-down"></i>
             </button>
             <ul class="dropdown-menu menu-right" role="menu">
-                {% if editable %}
+                {% if editable and not is_new %}
```

Comment:
Actually, disregard that entire comment. It might be simplest to lift the `editable` condition into its parent `if`:

```
if editable and not is_new
    Share
endif
Print
if editable and not is_current_conditions
    Rename
    Duplicate
    Delete
endif
```